src/settings/settingsdialog.cpp

```diff
@@ -310,6 +311,14 @@ void SettingsDialog::onOcrLanguagesPathChanged(const QString &path)
     ui->ocrLanguagesListWidget->addLanguages(Ocr::availableLanguages(path));
 }
 
+void SettingsDialog::selectOcrConfigPath()
+{
+    const QString path = ui->ocrConfigPathEdit->text();
+    const QString file = QFileDialog::getOpenFileName(this, tr("Select OCR config path"), path);
```

Comment:
I usually prefer to pass small thing directly, so just for consistency:
```suggestion
    const QString file = QFileDialog::getOpenFileName(this, tr("Select OCR config path"), ui->ocrConfigPathEdit->text());
```